modules/rte_mis_core/src/Batch/LocationTermBatch.php

```diff
@@ -299,10 +299,12 @@ public static function importMultiple(int $fileId, array $details, array &$conte
               $parentsTerm = reset($parentsTerm);
               if ($parentsTerm instanceof TermInterface) {
                 if ($j == 4 && $parentsTerm->id() == $lastNonEmptyParents[$j - 2]) {
-                  $context['results']['passed'][] = $value;
+                  // Do not track the existing term.
+                  // $context['results']['passed'][] = $value;.
                 }
                 elseif ($j != 4 && $parentsTerm->id() == $lastNonEmptyParents[$j - 1]) {
-                  $context['results']['passed'][] = $value;
+                  // Do not track the existing term.
+                  // $context['results']['passed'][] = $value;.
```

Comment:
Can we just remove this instead of commenting out.

Reply:
Updated